SPECS/mstflint/mstflint.spec

```diff
@@ -23,15 +23,35 @@ BuildRequires:  libxml2-devel
 BuildRequires:  make
 BuildRequires:  openssl-devel
 BuildRequires:  zlib-devel
+BuildRequires:  kernel-devel
+BuildRequires:  kmod
 Requires:       python3
+Requires:       kernel
+
+# set package name
+%{!?_name: %global _name mstflint}
+
+%global kver %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}-%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-devel))
+%global ksrc %{_libdir}/modules/%{kver}/build
+%global debug_package %{nil}
+%global kernel_source() %{ksrc}
+%global kernel_release() %{kver}
```

Comment:
Why create these 2 functions? Couldn't we just use the 2 macros defined above directly?

Reply:
@PawelWMS  I've tried to follow the original mstflint kernel spec as much as possible https://github.com/Mellanox/mstflint/blob/master/kernel/mstflint_kernel.spec